index.html

```diff
@@ -182,6 +182,19 @@ <h1 id="skill-text">Skills</h1>
             </div>
         </div>
     </section>
+
+    <div class="contact-me">
+        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <form action="https://formspree.io/f/xrgjdpow" method="POST">
+            <input type="text" name="name" class="text" placeholder="Full name" maxlength="30" required/>
+            <input type="email" class="text" placeholder="Email address" required/>
+            <textarea name="comment" class="input-text" placeholder="Enter text here" maxlength="500"></textarea>
+        <button type="submit" class="button">Get in touch</button>
```

Comment:
- [ ] (Optional) Kindly make the `textarea` input **_required_**. Remember that all input fields should be _**required**_ to activate the validation on submit. 👍